ddl/pausetest/pause_test.go

```diff
@@ -240,7 +241,7 @@ func TestPauseAndResumeMain(t *testing.T) {
 	cancelWhenReorgNotStart := atomicutil.NewBool(false)
```

Comment:
Do we need to check why `TestPauseAndResumeMain` takes so long time?

Reply:
+1. We should investigate it in details instead of just moving it.

Reply:
The purpose of migration is to isolate it from other tests. troubleshooting of the issue is not affected.